packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.html

```diff
@@ -4,28 +4,57 @@
-                               [descriptionPrimary]="legendSeries.legendDescriptionPrimary"
-                               [descriptionSecondary]="legendSeries.legendDescriptionSecondary"
-                               [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
-                               (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)"
-                               (mouseenter)="chartAssist.emphasizeSeries(legendSeries.id)"
-                               (mouseleave)="chartAssist.resetVisibleSeries()"
-                               [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state"
-                               (click)="onLegendClick(legendSeries, $event)">
+            <nui-legend-series 
+                        class="xy-chart__legend-series"
+                        [ngClass]="'xy-chart__legend-series--state-' + chartAssist.renderStatesIndex[legendSeries.id]?.state"
+                        *ngFor="let legendSeries of chartAssist.legendSeriesSet; trackBy: chartAssist.seriesTrackByFn"
+                        [isSelected]="!chartAssist.isSeriesHidden(legendSeries.id)"
+                        (isSelectedChange)="chartAssist.toggleSeries(legendSeries.id, $event)"
+                        (mouseenter)="chartAssist.emphasizeSeries(legendSeries.id)"
+                        (mouseleave)="chartAssist.resetVisibleSeries()"
+                        [seriesRenderState]="chartAssist.renderStatesIndex[legendSeries.id]?.state">
                 <nui-rich-legend-tile [value]="chartAssist.getHighlightedValue(legendSeries, 'y', undefined, valueAccessorKey)"
-                                      [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
-                                      [color]="chartAssist.palette.textColors.get(legendSeries.id)">
+                            [backgroundColor]="chartAssist.palette.standardColors.get(legendSeries.id)"
+                            [color]="chartAssist.palette.textColors.get(legendSeries.id)">
                     <nui-chart-marker rightEdge
-                                      [marker]="chartAssist.markers.get(legendSeries.id)"
-                                      [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
+                                [marker]="chartAssist.markers.get(legendSeries.id)"
+                                [color]="chartAssist.palette.standardColors.get(legendSeries.id)">
                     </nui-chart-marker>
                 </nui-rich-legend-tile>
+                <div description class="description-container d-flex flex-column nui-text-small justify-content-center">
+                        <div *ngIf="!legendSeries.link" 
+                            class="description description-primary" 
+                            [ngClass]="{'nui-text-link-small link': ((legendSeries?.link) || seriesInteractive) }"
```

Comment:
I just realized we don't need the check for `legendSeries?.link` here because `legendSeries?link` is always false for this element.

```suggestion
                            [ngClass]="{'nui-text-link-small link': seriesInteractive }"
```